authorization/v1/types.go

```diff
@@ -446,6 +446,7 @@ type ClusterRoleList struct {
 // subject are allowed in the namespace to which the RoleBindingRestriction
 // belongs.  If any one of those RoleBindingRestriction objects matches
 // a subject, rolebindings on that subject in the namespace are allowed.
+// +kubebuilder:singular=rolebindingrestriction
```

Comment:
I understand why `SecurityContextConstraints` needs the explicit singular, but not this one.

Reply:
Yeah the default should be fine for RBR

Reply:
Without the `singular` tag explicitly set, the controller-tools generate will not add a singular name tag (like this one: https://github.com/openshift/cluster-config-operator/blob/master/manifests/0000_03_authorization-openshift_01_rolebindingrestriction.crd.yaml#L11)

If we don't need that and the default is okay, I can remove it. Just double checking because that type is set as `rolebindingrestrictions.authorization.openshift.io` right now

Reply:
I bet kube-apiserver will just lower-case the kind name. That would be correct.

Reply:
Updated. This PR is now just adding the singular tag to SCC